sass/_theme_rst.sass

```diff
@@ -53,8 +50,41 @@
     margin-left: $base-line-height
     line-height: $base-line-height
     margin-bottom: $base-line-height
-  .literal-block, pre.literal-block
-    @extend .codeblock
+
+  pre.literal-block, div[class^='highlight']
+    border: 1px solid $table-border-color
+    padding: 0px
+    overflow-x: auto
+    line-height: normal
+    // 1px hack otherwise border won't show. lame
+    margin: 1px 0 $base-line-height 0
+    div[class^='highlight']
+      border: none
+      margin: 0
+  // Needs 100% width for line highlighting to work properly
+  div[class^='highlight'] td.code
+    width: 100%
+  .linenodiv pre
+    border-right: solid 1px lighten($table-border-color, 2%)
+    margin: 0
+    padding: $base-line-height / 2 $base-line-height / 2
+    font-family: $code-font-family
+    font-size: 12px
```

Comment:
Perhaps `font-size: $base-font-size * 0.75` ?
(Also in other places in this PR)

Reply:
This would be a new behavior correct? Seems fine but maybe this is better for another PR?

Reply:
The behavior would be the same (`$base-font-size` is 16px), but indeed perhaps better to move to a new PR that adds a `$code-font-size` in `_theme_variables.sass` so we can be sure we have consistent font sizes.